config/default-spinnaker-local.yml

```diff
@@ -110,9 +109,11 @@ providers:
     enabled: ${SPINNAKER_KUBERNETES_ENABLED:false}
 
     primaryCredentials:
-      name: my-docker-registry-account
-      address: https://index.docker.io/
-      repository: library/nginx
+      name: my-docker-registry
```

Comment:
I just noticed I made a typo at line 103. Also, there isnt a link to documentation url like the other sections. You might want to add one, but the docs are outside this PR.